### PR TITLE
fix!: remove image element proxy check

### PR DESCRIPTION
**Pull request by jlipps**
and allow proxyRouteIsAvoided to inspect the body (supports https://github.com/appium/appium/pull/15078)

### CODE REVIEW

File: lib/protocol/protocol.js [hunk 1]
```diff
@@ -10,25 +10,13 @@ import B from 'bluebird';
 import {
   formatResponseValue, formatStatus,
 } from './helpers';
-import {
-  MJSONWP_ELEMENT_KEY, W3C_ELEMENT_KEY, MAX_LOG_BODY_LENGTH, PROTOCOLS,
-  DEFAULT_BASE_PATH, IMAGE_ELEMENT_PREFIX,
-} from '../constants';
+import { MAX_LOG_BODY_LENGTH, PROTOCOLS, DEFAULT_BASE_PATH } from '../constants';
```
Comment (mykola-mokhnach) [inline lib/protocol/protocol.js:13]: do we still need the unused constants?

File: lib/basedriver/driver.js [hunk 2]
```diff
@@ -470,10 +470,11 @@ class BaseDriver extends Protocol {
    * should be made available to overridden methods.
    * @param {string} method - HTTP method of the route
    * @param {string} url - url of the route
+   * @param {object} body - webdriver request body
```
Comment (mykola-mokhnach) [inline lib/basedriver/driver.js:473]: I assume the type of body should be something like `{?*}` or it is always an object?

Comment (jlipps) [reply lib/basedriver/driver.js:473]: yeah good point

File: lib/basedriver/driver.js [hunk 3]
```diff
@@ -470,10 +470,11 @@ class BaseDriver extends Protocol {
    * should be made available to overridden methods.
    * @param {string} method - HTTP method of the route
    * @param {string} url - url of the route
+   * @param {*?} body - webdriver request body
```
Comment (mykola-mokhnach) [inline lib/basedriver/driver.js:473]: `{*?}` -> `{?*}`

**Review by mykola-mokhnach — APPROVED** [review]

File: lib/basedriver/driver.js (hunk 3 above)
Comment (jlipps) [reply lib/basedriver/driver.js:473]: oh, i thought optionality was designated after the type, probably thinking of some other language like typescript maybe.

**Comment by jlipps** [comment]
published as 2.0.0-beta.5

**Pull request merged**